dummy_data/dummy_data.json

```diff
@@ -1,7 +1,7 @@
 [
 {
   "model": "battDB.devicespecification",
-  "pk": 1,
+  "pk": 3,
```

Comment:
Do you generate this dummy data programatically? If so, how?

Reply:
The data was generated manually (using the frontend) but saved automatically using `python manage.py dumpdata`. See [this extra dummy_data README](https://github.com/ImperialCollegeLondon/Faraday-liionsden/tree/develop/dummy_data). 

I did investigate using model bakery (as used in tests) to generate dummy data but as it fills fields with random strings etc. it's not so easy to work with for frontend development!